packages/web-comment/src/routes/index.js

```diff
@@ -0,0 +1,38 @@
+const { readdirSync } = require('fs');
+
+// works line Next, urls come from dir names in routes director
```

Comment:
I think you dropped a y at the end!

Reply:
Good spot, spellchecker won't catch that one!